package.json

```diff
@@ -82,7 +82,11 @@
       "node tools/fetch-docs-links.js",
       "git add docs/src/_data/further_reading_links.json"
     ],
-    "docs/**/*.svg": "trunk check --fix --filter=svgo"
+    "docs/**/*.svg": "trunk check --fix --filter=svgo",
+    "docs/src/_data/{rules_meta,rule_versions}.json": [
+      "node tools/update-rule-type-headers.js",
+      "git add lib/types/rules/*.ts"
+    ]
```

Comment:
I believe that yorkie detects Jenkins as a CI/CD environment and thus doesn't install git hooks, so if we'd like to update rule type headers during releases, I think we should run the tool directly from Makefile.js rather than relying on lint-staged.

Reply:
I updated `Makefile.js` so the script will run on all rules during a release. This will update `@since` comments in TSDoc. I also changed the script so it will read rule metadata directly from the rule rather than using `docs/src/_data/rules_meta.json`.